llama-index-integrations/multi_modal_llms/llama-index-multi-modal-llms-openai/llama_index/multi_modal_llms/openai/utils.py

```diff
@@ -38,7 +38,10 @@ def generate_openai_multi_modal_chat_message(
     completion_content = [{"type": "text", "text": prompt}]
     for image_document in image_documents:
         image_content: Dict[str, Any] = {}
-        mimetype = image_document.image_mimetype or "image/jpeg"
+        if isinstance(image_document, ImageDocument):
```

Comment:
If the input type is `ImageDocument`, then we should only be passing in `ImageDocument` objects

Something else is passing in other object types, like `NodeWithScore`, and we should fix that instead?